.github/workflows/publish.yml

```diff
@@ -22,4 +17,6 @@ jobs:
         - name: Use Node.js
           uses: actions/setup-node@v1
         - name: publish to master
-          run: bash ./scripts/publish.sh        
\ No newline at end of file
+          run: bash ./scripts/publish.sh $GIT_BOT_TOKEN
```

Comment:
nice. we could also just get content of publish.sh and put it in here. what do you think?

Reply:
yeah, that would work too. Maybe better and then have it all in one single place?